### ISSUE TITLE
Request: Option to start stacer maximized

### ISSUE THREAD

**Issue by tester1969** [oguzhaninan/Stacer#369]
Either in command line or some radio button. *Please*

**Comment by alexmyczko** [comment]
What is your window manager or desktop environment?

For me, Window Maker, stacer starts in maximized mode, or you mean the application all full screen application?

**Comment by tester1969** [comment]
Fedora Gnome Xorg. I auto start stacer and I want when started to start all full screen, covers all screen. 
Actually what I have done is to install a 7" 2nd monitor on my case and I want to have permanently there show stacer clocks :) So after each reboot I send there stacer but I want to cover the whole 7" screen.

**Comment by alexmyczko** [comment]
Since it is qt you could try doing what is there https://stackoverflow.com/questions/19817881/qt-fullscreen-on-startup

or try if qt accepts the x11 standard command for applications https://stackoverflow.com/questions/36864183/how-to-i-set-the-default-startup-window-size-for-an-x11-application-such-as-xmg

**Comment by tester1969** [comment]
Ah thanx, I am using geometry to send to 2nd display, I ll try to setup size with that. If it doesn't work I ll try to edit the code as per link you gave me.

**Comment by tester1969** [comment]
Yes it worked, put the resolution as two first parameters of --geometry and it worked. I put 
--geometry 1280x1024+3840+0 , since the resolution of 7" screen is 1280x1024 and the 1st monitor analysis horizontally is 3840
